src/switch_core_state_machine.c

```diff
@@ -390,6 +389,15 @@ static void switch_core_standard_on_execute(switch_core_session_t *session)
 
 	}
 
+	post_dialplan_execute = switch_channel_get_variable(session->channel, "post_dialplan_execute");
```

Comment:
What happen when I execute `uuid_broadcast`, will it call the post dialplan function?

Reply:
i think it should, since this happens in _on_execute handler

Reply:
Ideally this should only be executed only when processing dialplan commands during `ROUTE` state. Otherwise CC commands which always call `uuid_broadcast` will execute the post dialplan commands.

Reply:
ok, I think I understand what you mean now.. I moved it to `on_routing` handler